packages/arb-token-bridge-ui/src/hooks/useRedeemRetryable.ts

```diff
@@ -73,21 +70,22 @@ export function useRedeemRetryable(): UseRedeemRetryableResult {
         resolvedAt: isSuccess ? dayjs().valueOf() : null,
         depositStatus: isSuccess ? DepositStatus.L2_SUCCESS : tx.depositStatus
       })
+
+      setIsRedeeming(false)
+
+      // track in analytics
+      trackEvent('Redeem Retryable', { network: l2NetworkName })
     } catch (error: any) {
+      setIsRedeeming(false)
+
```

Comment:
NIT: if we want to do something when the code is working or not, we can use `finally` instead